### DISCUSSION TITLE
X Axis problem

### DISCUSSION THREAD

**Discussion by filap** [arkypita/LaserGRBL#961] [Q&A]
Hi I am using LaserGRB 3.1.7; 3.2.0, 3.3.0 with arduino uno and software 1.1.  

And I have strange problems with x axis, it looks like during laser engraving (movement not engraving) system is loosing orientation and what is printed is not what is in the g-code file.

This is not hardware problem because when I change (dir/step) between x i y this problem is visible on Y axis  

have anyone faced this problem ?

regards

**Comment by gmmanonymus111** [comment]
Maybe your motors are losing steps. What kind of power supply do you have?

> This is not hardware problem because when I change (dir/step) between x i y this problem is visible on Y axis

This also could mean a config issue, could you post your `$$` (GRBL) settings?
